pvellanki@qa2434:~/orsa$ python3 test.py  orsa_postmove_2023-06-28.xml postmove.db
Traceback (most recent call last):
  File "/home/pvellanki/orsa/test.py", line 53, in <module>
    cursor = conn.execute(count_query)
sqlite3.DatabaseError: file is not a database
